Added multicast and ICCRG (#2)
chris-wood/ietf-masque: proposed_charter.md

> The primary goal of this working group is to develop mechanisms that allow configuring and concurrently running multiple proxied stream- and datagram-based flows inside a HTTP/3 connection. The group will specify HTTP or HTTP/3 extensions to enable this functionality within an HTTP/3 connection. The group will focus on a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying. Specifying proxy server discovery mechanisms is out of scope for the group. However, the group may specify techniques for identifying proxy servers to aid future discovery mechanisms.
> 
> Support for UDP and IP multicast use cases is out of scope, but the working group is encouraged to make the mechanisms extensible for multicast, if practicable.
> 

Reviewer: ```suggestion
The primary goal of this working group is to develop mechanisms that allow configuring and concurrently running multiple proxied stream- and datagram-based flows inside a HTTP/3 connection. The group will specify HTTP or HTTP/3 extensions to enable this functionality within an HTTP/3 connection. The group will focus on a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying. Multicast UDP and multicast IP support is out of scope. However, the group may specify extension points that would enable future work on multicast. Specifying proxy server discovery mechanisms is also out of scope. However, the group may specify techniques for identifying proxy servers to aid future discovery mechanisms.
```
